SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/AlwaysInterestedTest/AlwaysInterestedTest.cpp

```diff
@@ -73,24 +74,19 @@ void AAlwaysInterestedTest::PrepareTest()
 
 			if (HasAuthority())
 			{
-				ActorWithAlwaysInterestedProperty =
-					GetWorld()->SpawnActor<AAlwaysInterestedTestActor>(LocalWorkerPosition, FRotator::ZeroRotator, FActorSpawnParameters());
+				ActorWithAlwaysInterestedProperty = SpawnActor<AAlwaysInterestedTestActor>(LocalWorkerPosition, FRotator::ZeroRotator);
 
-				InterestedInThisReplicatedActor =
-					GetWorld()->SpawnActor<ASmallNCDActor>(LocalWorkerPosition, FRotator::ZeroRotator, FActorSpawnParameters());
+				InterestedInThisReplicatedActor = SpawnActor<ASmallNCDActor>(LocalWorkerPosition, FRotator::ZeroRotator);
 
-				NotInterestedInThisReplicatedActor =
-					GetWorld()->SpawnActor<ASmallNCDActor>(LocalWorkerPosition, FRotator::ZeroRotator, FActorSpawnParameters());
+				NotInterestedInThisReplicatedActor = SpawnActor<ASmallNCDActor>(LocalWorkerPosition, FRotator::ZeroRotator);
 
 				// This actor is used later as a replacement for InterestedInThisReplicatedActor, so isn't immediate added to
 				// AlwaysInterested
-				OtherInterestedInThisReplicatedActor =
-					GetWorld()->SpawnActor<ASmallNCDActor>(LocalWorkerPosition, FRotator::ZeroRotator, FActorSpawnParameters());
```

Comment:
Is there a reason we were passing `FActorSpawnParameters()` every time?

Reply:
Maybe it wasn't always a default parameter like it is now? This change will provide exactly the same functionality for less typing so seems good to me.

Reply:
ah yea i guess it could have changed